tests/zkevm/test_worst_compute.py

```diff
@@ -90,7 +92,7 @@ def test_worst_keccak(
 
     tx = Transaction(
         to=code_address,
-        gas_limit=gas_limit,
+        gas_limit=env.gas_limit,
         gas_price=10,
```

Comment:
Can we take the opportunity to remove "gas_price" from this and next test?

Reply:
Yep, I'm doing this in an [upcoming PR](https://github.com/ethereum/execution-spec-tests/pull/1611/files#diff-cdcef4e8328e54064d53a63c2653db5f0eeab8030d09db345b118bcd87b0cf50L39) which has other changes.

All `gas_limit` parametrizations will be gone.

Reply:
In your honor, I'll do it this one in particular you pointed out in this PR :D